src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -71,9 +71,7 @@ public struct MetricPoint
             }
         }
 
-        public string[] Keys { get; internal set; }
-
-        public object[] Values { get; internal set; }
+        public MetricPointAttributes Attributes { get; }
```

Comment:
The API use the word 'Tags' instead of Attributes, so we can stick with same name here too.

Reply:
I renamed the property "Tags" and then I renamed the structure `ReadOnlyTagCollection`. There isn't anything really MetricPoint-specific about it, so I thought it would be good to keep it general in case we wanted to use it in other spots in the future?